src/app/pages/back-to-office/back-to-office.actions.ts

```diff
@@ -0,0 +1,9 @@
+import { createAction } from '@ngrx/store';
+
+export const BackToOfficeViewDidEnter = createAction(
+  '[BackToOfficePage] ID entered',
```

Comment:
Can we make the description more relevant to the action, in DES 3 this action description is = 
```'[BackToOfficePage] BackToOffice view did enter'```